app/packs/src/apps/mydb/elements/details/reactions/variationsTab/ReactionVariationsCellComponents.js

```diff
@@ -21,13 +21,13 @@ function RowToolsCellRenderer({
   const { reactionShortLabel, copyRow, removeRow } = context;
   return (
     <div>
-      <Badge>{getVariationsRowName(reactionShortLabel, variationsRow.id)}</Badge>
+      <Badge bg='secondary' className='me-2'>{getVariationsRowName(reactionShortLabel, variationsRow.id)}</Badge>
       {' '}
```

Comment:
You already apply a spacing class to Badge increase it if you need extra space. Better yet make the enclosing div responsible for handling space between elements.